### ISSUE TITLE
Add more emotions

### ISSUE THREAD

**Issue by n4sr** [NimaKhajehpour/MyMood#35]
Currently it seems to track 5 different levels of either positivity or negativity. Being able to select an emotion from a list as well as recording the reason behind the emotion would be a great enhancement to this project.

![emotion_wheel2_colour.png](https://github.com/NimaKhajehpour/MyMood/assets/54733427/8caef223-097e-455c-a6e9-3f1b858e57dd)

**Comment by NimaKhajehpour** [comment]
Hey there, thanks for the issue. i know its been a long time since you made this issue and im sorry for the delay. i am resuming the updates and maintaining the app.
i agree with you mood monitoring has a lot of potential to make it even wider. i will look into it 😁
oh also have i mentioned that i am fully re-designing the app and adding an api to it so that its an online application?
i also made a discord community would love to see you there too.
you can find the link in my github bio.😉

**Comment by gnu1dea** [comment]
Hi Nima,
I'm so happy this app is still being maintained: thank you very much!

Will there also remain an option to use it _offline_ & make local backups?

**Comment by NimaKhajehpour** [comment]
hey @gnu1dea im so happy to see that users didn't lose their hope on me and my apps and you are still waiting for me to maintain the application.
im so thankful for that.
as for keeping the option of saving everything offline, the main idea of making it online was to get around the challenges of the offline data saving and the way backing everything up worked and was a little hard and challenging.
but i guess if there is enough demand on user base to have the option in the app we shall have it.
but i need to be more in touch with the community and have interactions.

**Comment by NimaKhajehpour** [comment]
the other problem for me is that the community doesn't exist. tried to make community and users mostly dont join it so for that reason i dont know exactly what is mostly wanted by the users and what is not, so its a little hard to make a communication with my users.

**Comment by n4sr** [comment]
> the other problem for me is that the community doesn't exist. tried to make community and users mostly dont join it so for that reason i dont know exactly what is mostly wanted by the users and what is not, so its a little hard to make a communication with my users.

I understand that finding a way to get feedback or direction for your project is difficult when there is a very small user base. I do not have an answer for how to form that community you are seeking.

You could open up a new issue and pin it for discussion regarding whether you should add a feature or not. You might be able to get feedback that way.

**Comment by NimaKhajehpour** [comment]
yeah users don't seem to look at that either, the best way of gathering users was to add the addresses to the app so that they can see it and join it. i guess i should solve it somehow to get more feedback.

**Comment by NimaKhajehpour** [comment]
and also you didnt say if i put an option for using it online or offline would you use the app?
@n4sr

**Comment by n4sr** [comment]
> and also you didnt say if i put an option for using it online or offline would you use the app.

I do not have enough information to make that judgment. It depends on how it is implemented and how it affects functionality. I am not sure what parts of this app you intend to move server side and what the reasoning behind doing so would be.

There are many mood tracker apps out there. Some require an online account on their website, display advertisements in the app, and collect data in the hopes of selling it off later. I personally use your app because it allows me to log my mood and what is affecting it without having to deal with creating an account, viewing advertisements, or having my data sold. I like that I do not have to give this app network access in order for it to function. If that changes then I would have no reason to use this anymore.

Ultimately this is your software, it is your decision to do with it what you desire.

**Comment by n4sr** [comment]
@NimaKhajehpour The bug/crash/feature reporting inside the app could be a good use of an API. As long as you give the user the option to not report a crash if they choose to do so. Some stack traces can include personally identifying information, so users need to be made aware of that and given the option of not submitting it if they choose.

I would avoid storing user data on servers at all. I cannot think of a good reason to do that, but I can come up with plenty of reasons not to.
